app/controllers/shelters_controller.rb

```diff
@@ -46,6 +46,5 @@ def update
 
   def pets
     @shelter = Shelter.find(params[:id])
-    @shelter_pets = @shelter.pets
```

Comment:
I removed the shelter_pets IV because I can just call `@shelter.pets` in the view